### PR TITLE
[plugin.video.southpark_unofficial] v0.5.8

### PR DESCRIPTION
**Pull request by wargio**
### Description
Matrix only release.
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.southpark_unofficial/addon.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<addon id="plugin.video.southpark_unofficial" name="South Park" version="0.5.8" provider-name="Deroad">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <summary lang="en">South Park Unofficial Player</summary>
```
Comment (enen92) [inline plugin.video.southpark_unofficial/addon.xml:11]: en_GB

File: plugin.video.southpark_unofficial/addon.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <language>en</language>
```
Comment (enen92) [inline plugin.video.southpark_unofficial/addon.xml:12]: en_GB

File: plugin.video.southpark_unofficial/addon.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        <description lang="en">Watch South Park episodes. The supported countries are the one that can view videos from https://southpark.cc.com, https://southparkstudios.co.uk, https://southparkstudios.nu or https://www.southpark.de.</description>
+        <disclaimer lang="en">Some parts of this addon may not be legal in your country of residence - please check with your local laws before installing.</disclaimer>
+        <license>GPL-2.0</license>
```
Comment (enen92) [inline plugin.video.southpark_unofficial/addon.xml:15]: GPL-2.0-or-later (or GPL-2.0-only) depending on your license (spdx identifier)

File: plugin.video.southpark_unofficial/addon.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        <source>https://github.com/wargio/plugin.video.southpark_unofficial</source>
+        <forum>https://forum.kodi.tv/showthread.php?tid=224662</forum>
+        <email>wargio at libero dot it</email>
+        <website>https://southpark.cc.com</website>
+        <reuselanguageinvoker>true</reuselanguageinvoker>
+        <assets>
```
Comment (enen92) [inline plugin.video.southpark_unofficial/addon.xml:21]: maybe also define the `<news>` tag with the news of your latest release. Unlike the changelog, this is shown in the interface and addon website

File: plugin.video.southpark_unofficial/default.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import socket
+import sys
+import southpark
+
+socket.setdefaulttimeout(30)
+plugin = southpark.SouthParkAddon(argv=sys.argv, last_season=23)
+plugin.handle()
+plugin = None
```
Comment (enen92) [inline plugin.video.southpark_unofficial/default.py:10]: line missing at the end of the file (not mandatory)

File: plugin.video.southpark_unofficial/addon.xml (hunk 4 above)
Comment (wargio) [reply plugin.video.southpark_unofficial/addon.xml:21]: not now. maybe i'll add in the future.

**Comment by wargio** [comment]
I've adjusted the plugin as you have asked @enen92

**Pull request merged**